siphon/simplewebservice/mesowest.py

```diff
@@ -0,0 +1,124 @@
+import warnings
+
+from bs4 import BeautifulSoup
+import numpy as np
+import pandas as pd
+
+from ..http_util import HTTPEndPoint
+
+warnings.simplefilter("ignore")
+
+class MesoWest(HTTPEndPoint):
+    """Download and parse data from the University of Utah's MesoWest archive."""
+    def __init__(self):
+        """Set up endpoint."""
+        super(MesoWest, self).__init__('https://mesowest.utah.edu/cgi-bin/droman/')
```

Comment:
Using this as the API endpoint seems...sketchy? Can you point to somewhere that says this is a good endpoint to use?

Reply:
It is a bit odd that the archive it setup like this. I am running some tests to see if the https://mesowest.utah.edu/ endpoint works, but the cgi-bin/droman/ part will likely end up moved down to the path variable in the  _get_data_raw function.